src/Pages/form/Components/StrokePredictionForm.tsx

```diff
@@ -6,11 +6,39 @@ import SectionTitle from "../Components/SectionTitle";
 import EKG12LeadsSection from "../Components/EKG12LeadsSection";
 import UnderLyingDiseaseSection from "../Components/UnderLyingDiseaseSection";
 import VitalSignsSection from "../Components/VitalSignsSection";
+import SideBarProgress from "../Components/SideBarProgress";
 import { Formik, Form } from "formik";
 import validations from "../Validation/validations";
 import { Button } from "@material-ui/core";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import { Section, ScrollingProvider } from "react-scroll-section";
+
+const useStyle = makeStyles((theme) =>
+  createStyles({
+    sidebar: {
+      position: "fixed",
+      top: "1rem",
+      width: "300px",
+      right:"50px",
```

Comment:
Please handle responsively. You can try to decrease the screen and it should be still good.